server/welcomebot.go

```diff
@@ -192,7 +198,7 @@ func (p *Plugin) processActionMessage(messageTemplate MessageTemplate, action *A
 
 	tmpMsg, _ := template.New("Response").Parse(strings.Join(configMessageAction.ActionSuccessfulMessage, "\n"))
 	var message bytes.Buffer
-	tmpMsg.Execute(&message, messageTemplate)
+	_ = tmpMsg.Execute(&message, messageTemplate)
```

Comment:
And here

Reply:
Ofc, I should have thought about it.Done in [`166ff26`](https://github.com/mattermost/mattermost-plugin-welcomebot/pull/51/commits/166ff263682e7619e2859f9436c829a38de9698c)